workflows/plots.py

```diff
@@ -81,7 +81,7 @@ def gather_inference_results(output_dir, demog, output, method, chrm_mask,
                 annot_mask_i = annot
             if method == "stairwayplot":
                 nt = pd.read_csv(infile, sep="\t", skiprows=5)
-                nt.columns = nt.columns.str.replace('[%,.]','')
+                nt.columns = nt.columns.str.replace('%','').str.replace('.', '').str.replace(',', '')
```

Comment:
The original simply didn't do the subs in my environment and kept the original strings unchanged. This works, but is more long-winded.